scripts/deploy_mvn.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+echo $GPG_SECRET_KEY | base64 --decode | $GPG_EXECUTABLE --import || true
+echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust || true
+mvn --settings scripts/mvn-release-settings.xml package gpg:sign deploy -Prelease -DskipTests -B -U;
```

Comment:
if your merge is done and builds correctly I will try to build a release again, just to make sure still everything is working...